### PR TITLE
Criteria dings

### LINKED ISSUE #5308: Add "privacy ding" to criteria sections
- [x] Add mini privacy ding when applicable (i.e. when the answer is `Yes` to the ding questions related to "How does it use this data?","How can you control your data?","What is the company’s known track record of protecting users’ data?","Does this product meet our Minimum Security Standards?") - [icon](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)

This requires criteria sections are implemented and updated first.

### Criteria section desktop
image

### Criteria section mobile
image

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5308 

Bonus: I also fixed mobile styling for the first 3 questions in the Privacy section (`Can it snoop on me?`, `What is required to sign up?`, and `What data does it collect?`)

- https://foundation-s-issue-5308-f6c624.herokuapp.com/en/privacynotincluded/products/percy-cypress/
- edit page: https://foundation-s-issue-5308-f6c624.herokuapp.com/cms/buyersguide/generalproduct/edit/1/

### CODE REVIEW

File: source/sass/buyers-guide/views/product.scss [hunk 1]
```diff
@@ -129,13 +129,6 @@ $pni-product-breakpoint-larger: $bp-md;
       }
     }
   }
-
-  .it-uses,
-  .signup-requirement {
-    .separator {
-      color: $gray-20;
```
Comment (mmmavis) [inline source/sass/buyers-guide/views/product.scss:136]: we no longer have `.separator` elements

**Comment by Pomax** [comment]
@mmmavis is this still WIP?

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 2]
```diff
@@ -151,15 +151,15 @@ <h3 class="pni-product-smaller-body-bold mb-2">{% trans "What data does it colle
 
 
           {% trans "How does it use this data?" as how_does_it_use_data_collected %}
-          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  %}
+          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  ding_control=product.data_collection_policy_is_bad %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:154]: Hm, can we change this name to `show_ding` or even just `ding`? A controller is typically far more elaborate than a boolean.

**Review by Pomax — CHANGES_REQUESTED** [review]

**Comment by mmmavis** [comment]
yes this is still WIP

**Comment by Pomax** [comment]
I guess the question is whether it matters to expose it, since we already have the "warning: privacy not included" text bar. This feels like it's more "decoration", and that warning bar is the real information.

**Comment by mmmavis** [comment]
@Pomax I checked with Kristina and Sabrina and they both thought the dings are more than just decorative. 

I went with adding `aria-label="This  section requires attention."` to the `<section>` of the relavent criteria because it can signal attention on the "section" overview level.  We could add an ` ding` within the `<section>` but assistive technology users will have to get into each section in order to know which specific criteria section requires extra attention, which is probably not ideal.

**Review by kristinashu — APPROVED** [review]
Nice! Design looks good!

**Review by Pomax — APPROVED** [review]

**Pull request merged**